README.md

```diff
@@ -56,7 +56,7 @@ mvn package
 
 Then manually install the following JARs:
 
-* target/client-java-api-4.0.0-beta1-SNAPSHOT.jar
+* target/client-java-api-5.0.0.jar
```

Comment:
if you build it yourself the beta and snapshot will be there...

Reply:
fixed